test/pylib/scylla_cluster.py

```diff
@@ -1776,6 +1777,45 @@ async def _server_get_process_status(self, request: aiohttp.web.Request) -> Opti
         server_id = ServerNum(int(request.match_info["server_id"]))
         return self.cluster.server_get_process_status(server_id)
 
+async def get_scylla_2025_1_executable() -> pathlib.Path:
+    async def run_process(cmd, **kwargs):
```

Comment:
Let's push this out of the series. It requires a little more review and has a completely different set of requirements. For example, how to we update releases as patch releases advance and as we generate new major releases.

I'm not against it, but it's out of place here.